README.md

```diff
@@ -20,3 +22,103 @@ When it gets included in your project it should automatically build the browser
    - you can either manually edit the changelog or use lerna-changelog to generate it based on the merged PRs (`GITHUB_AUTH=your-token npx lerna-changelog`).
 3. release-it pushes the tag to GitHub
 4. Woodpecker will publish the new version to npm
+
+## API
```

Comment:
I would move this above the development and releasing sections, this info seems more relevant for most people.